packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -5,15 +5,33 @@ public struct MediaInfo: Encodable {
     public let title: String?
     public let caption: String?
     public let alt: String?
+    public let metadata: Encodable
 
-    public init(id: Int32?, url: String?, type: String?, caption: String? = nil, title: String? = nil, alt: String? = nil) {
+    private enum CodingKeys: String, CodingKey {
+        case id, url, type, title, caption, alt, metadata
+    }
+
+    public init(id: Int32?, url: String?, type: String?, caption: String? = nil, title: String? = nil, alt: String? = nil, metadata: Encodable? = nil) {
```

Comment:
This change shouldn't impact the iOS app as we set a default value for the new parameter `metadata`.